easybuild/easyconfigs/x/xcb-proto/xcb-proto-1.11-intel-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'xcb-proto'
+version = '1.11'
+
+homepage = 'http://xcb.freedesktop.org/'
+description = """The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint,
+latency hiding, direct access to the protocol, improved threading support, and extensibility."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = ['http://xcb.freedesktop.org/dist/']
+sources = [SOURCELOWER_TAR_GZ]
+
+python = 'Python'
+pyver = '2.7.11'
+versionsuffix = '-%s-%s' % (python, pyver)
+builddependencies = [(python, pyver)]
```

Comment:
I think it is better to add `Python` as a dep, and not a system `Python`. It works with both `Python2` and `Python3`, so that should not be a problem. I am also thinking to add a `Pythonless` version, if xcb-proto is a builddep, and `Python` is a dep for a program. (I am working on Python with X.)

I am not 100% sure that the same scripts installed with `Python2` and `Python3`

What do you think? Should we still use 
```
allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
```